README.md

```diff
@@ -1,6 +1,17 @@
 # Simulink Models
 
 ## Folder structure
-- models: simulink models
-- config: shared configuration files
+- autogen: contains the Autogen framework
+- models: simulink model projects utilizing the Autogen framework
+- config: path configuration files for the Autogen framework
 - test_signals: audio files for testing the models
+
+
+## Getting started with a new Simulink Model
+- Create the Simulink Model matching Autogen examples  
```

Comment:
What does this mean exactly? Copying an existing example slx file?

Reply:
Ideally we have a template they can use at some point, but for now I kept it vague because we don't really have it defined what that is yet, in particular with noise suppression changes.